tests/e2e/hcp_cluster_test.go

```diff
@@ -340,4 +365,57 @@ var _ = Describe("HCP cluster testing",
 				Expect(clusterDesc.OperatorIAMRoles).To(HaveEach(MatchRegexp("arn:aws:iam::[0-9]{12}:role/.+")))
 				Expect(clusterDesc.State).To(Equal(constants.Ready))
 			})
+
+		It("ROSA CLI cluster creation should show install/uninstall logs - [id:75534]",
+			labels.Critical,
+			labels.Runtime.Day1Supplemental,
+			func() {
+				clusterName := helper.GenerateRandomName("ocp-75534", 2)
+				flags, err := profilehandler.GenerateClusterCreateFlags(customProfile, rosaClient)
+				Expect(err).ToNot(HaveOccurred())
+
+				command := "rosa create cluster --cluster-name " + clusterName + " " + strings.Join(flags, " ")
+				rosalCommand := config.GenerateCommand(command)
+
+				stdout, err := rosaClient.Runner.RunCMD(strings.Split(rosalCommand.GetFullCommand(), " "))
+				Expect(err).To(BeNil())
+				Expect(stdout.String()).To(ContainSubstring(fmt.Sprintf("Cluster '%s' has been created", clusterName)))
+
+				By("wait for 5 minutes to install logs show up")
+				time.Sleep(5 * time.Minute)
```

Comment:
Wouldn't it be better to wait for the `installing` state of the cluster ?

Reply:
i tried 
```
err = clusterService.WaitClusterStatus(clusterID, "installing", 5, 24)
Expect(err).To(BeNil(), "It met error or timeout when waiting cluster to installing status")
```
but constantly facing error
`It met error or timeout when waiting cluster to installing status`

but at the same time, if we track the cluster status, the cluster gets installed successfully without any issues

Reply:
@AkashKanni Let's keep it that way for now.
Could you create a follow-up card to review this later ?

Reply:
Hey @radtriste i have created a ticket here https://issues.redhat.com/browse/OCM-11613, PTAL Thanks!